src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -636,11 +630,6 @@ class _NnMaterialLocalizationsDelegate extends LocalizationsDelegate<MaterialLoc
         // for 'en_US' instead.
         decimalFormat: intl.NumberFormat('#,##0.###', 'en_US'),
         twoDigitZeroPaddedFormat: intl.NumberFormat('00', 'en_US'),
-        //...
```

Comment:
Same question about removing the closing brackets

Reply:
I've been trying to improve this a little bit more but it's very complicated to extract the section of code from the `_NnMaterialLocalizationsDelegate`, so I thought also that it would be beneficial to show the whole class instead instead of doing a very complex RegEx replace. What do you think?